cli/test/e2e/package_install_test.go

```diff
@@ -255,4 +255,21 @@ key2: value2
 		})
 		require.Contains(t, err.Error(), "not found")
 	})
+
+	logger.Section("package installed update with missing old version", func() {
+		_, err := kappCtrl.RunWithOpts([]string{"package", "installed", "create", "--package-install", pkgiName,
+			"-p", packageMetadataName, "--version", packageVersion1,
+			"--values-file", "-"}, RunOpts{StdinReader: strings.NewReader(valuesFile1)})
+		require.NoError(t, err)
```

Comment:
We won't need this check because control won't get here if there is an error

Reply:
Done